<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTestDialog</class>
 <widget class="QDialog" name="dlgTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlgTestDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonGray">
       <property name="text">
        <string>开始生成灰度图</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar1">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar2">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="progressBar3">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progressBar4">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QProgressBar" name="progressBar5">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QProgressBar" name="progressBar6">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QProgressBar" name="progressBar7">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QProgressBar" name="progressBar8">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QProgressBar" name="progressBar9">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QProgressBar" name="progressBar10">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
